src/main/scala/firrtl/Compiler.scala

```diff
@@ -8,7 +8,8 @@ import annotations._
 
 import firrtl.ir.Circuit
 import passes.Pass
-import Utils.throwInternalError
+import firrtl.Utils.error
+import firrtl.Utils.throwInternalError
 
```

Comment:
Perhaps make these imports a bit more consistent: (this suggestion adds firrtl to all the associated lines but could also drop everywhere since in subpackage of firrtl)

import firrtl.annotations._  // Note that wildcard imports are not great....
import firrtl.ir.Circuit
import firrtl.passes.Pass
import firrtl.Utils.{error, throwInternalError}